Wangscape/tilegen/partition/TilePartitionerNoise.cpp

```diff
@@ -43,6 +44,19 @@ noise::module::ModulePtr TilePartitionerNoise::makeCornerModule(const Corners& c
     combiner.setInputModuleSource(2, central.getOutputModule());
     if (mDebugOutput)
```

Comment:
We can move it to a separate function, but the function would have `corners, left, top, central, border_h, border_v, combiner` as parameters. Splitting it up might be more feasible. `write_map` and `write_group` could be moved into private class methods. The same can be done for the folder/filename generators and the console IO. The block would probably manageable after that.

I don't think it's worthwhile moving this into a separate module; this is a temporary solution until #39 is implemented.